src/lib/leaflet-directive.js

```diff
@@ -1,4 +1,4 @@
-import layerFactory from './leaflet-utils/layer-factory'
+import createLayer from './leaflet-utils/layer-factory'
```

Comment:
Should file also be renamed?

Reply:
The file exposes several functions,  the default is called `createLayer`.  It also creates clusters so for now it's ok...